UI/cmake/feature-youtube.cmake

```diff
@@ -11,3 +11,10 @@ else()
   set(YOUTUBE_SECRET_HASH 0)
   set(YOUTUBE_CLIENTID_HASH 0)
 endif()
+
+if(YT_WEBAPP_ADDRESS)
+  target_sources(obs-studio PRIVATE window-dock-youtube-app.cpp window-dock-youtube-app.hpp)
+  target_enable_feature(obs-studio "YouTube WebApp Dock" YT_WEBAPP_ENABLED)
+else()
+  target_disable_feature(obs-studio "YouTube WebApp Dock")
+endif()
```

Comment:
Thank you for this comment.
I rechecked YOUTUBE_ENABLED and YT_WEBAPP_ENABLED (renamed to the YOUTUBE_WEBAPP_ENABLED), added dependency in the cmake files (new one and legacy).
Changed double "#if YOUTUBE_ENABLED && YOUTUBE_WEBAPP_ENABLED" to the single "#if YOUTUBE_WEBAPP_ENABLED".

Reply:
2) Yes. This code uses YT api (via youtube-api-wrappers.cpp) and definitely requires YOUTUBE_ENABLED.

1) I think yes.
  * We need a flag to enable or disable this yt dock functionality. So we need some cmake variable.
  * I'm not sure that this URL remain same in the feature (it changed in the past, it may change in the feature), like YOUTUBE keys.

So, I think, it's good to combine both variables (url and flag) into one, url. And use this url as a flag.

Reply:
Done.